### PR TITLE
Bump version to 0.12.0

### PR DESCRIPTION
**Pull request by camillebaldock**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 0.12.0
+
+Blank line parsing: parse new lines and maintain number of new lines in markdown
+instead of treating them as separtors.
+
+`===` separators for front matter.
```
Comment (dsingleton) [inline CHANGELOG.md]: This'll need to be updated in all Smartdown files using this new version right?

Comment (camilleldn) [inline CHANGELOG.md]: yes

Comment (camilleldn) [inline CHANGELOG.md]: hence my question about major bumps earlier

Comment (dsingleton) [inline CHANGELOG.md]: While pre `1.0` we've been using `Minor` bumps as equivalents to `Major`, certainly we've done the same for other syntax breaking changes in the past, and i'm happy with that. Once we do hit `1.0` then we'll follow semver properly.

**Pull request merged**